### PR TITLE
add Earliest and Latest reset options

### PR DESCRIPTION
**Pull request by SaiPrasanna9**

### CODE REVIEW

File: cmd/topicctl/subcmd/reset.go [hunk 1]
```diff
@@ -43,11 +45,38 @@ func init() {
...
+		"to-latest",
+		false,
+		"Resets offsets of consumer group members to latest offsets of partitions")
+
 	addSharedFlags(resetOffsetsCmd, &resetOffsetsConfig.shared)
 	RootCmd.AddCommand(resetOffsetsCmd)
 }
 
 func resetOffsetsPreRun(cmd *cobra.Command, args []string) error {
+	msg := "You must choose only one of the following reset-offset specifications: --toEarliest, --toLatest, --offset.  "
```
Comment (code-haven) [inline cmd/topicctl/subcmd/reset.go:64]:
```suggestion
	msg := "You must choose only one of the following reset-offset specifications: --to-earliest, --to-latest, --offset.  "
```

File: cmd/topicctl/subcmd/reset.go [hunk 2]
```diff
@@ -33,7 +35,7 @@ func init() {
 	resetOffsetsCmd.Flags().Int64Var(
 		&resetOffsetsConfig.offset,
 		"offset",
-		-2,
+		-5,
```
Comment (code-haven) [inline cmd/topicctl/subcmd/reset.go:38]: is this change necessary?

File: pkg/groups/groups.go [hunk 3]
```diff
@@ -192,3 +192,29 @@ func ResetOffsets(
...
 }
+
+// getOffset gets offset for a given topic partition for resetting offsets for consumer group
+func GetOffset(
+	ctx context.Context,
+	connector *admin.Connector,
+	topic string,
+	strategy string,
+	partition int,
+	offset int64,
```
[the remark this answers is not in the document]
Comment (SaiPrasanna9) [reply pkg/groups/groups.go]: removed offset param from this function, used an if condition which checks if a strategy is specified before calling this function, otherwise uses offset provided.

File: pkg/groups/groups_test.go [hunk 4]
```diff
@@ -114,6 +114,55 @@ func TestGetLags(t *testing.T) {
 	}
 }
 
+func TestGetOffset(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (SaiPrasanna9) [reply pkg/groups/groups_test.go:117]: updated TestResetOffsets with test cases to test new reset strategies.
